src/admin/controller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Admintask
+ * @author     Tasolglobal <testing@tasolglobal.com>
+ * @copyright  2016 Tasolglobal
```

Comment:
Change from `@copyright  2016 Tasolglobal` to `@copyright  __COMPONENT_COPYRIGHT__` in all files.